riak_test/oprga_test.erl

```diff
@@ -0,0 +1,92 @@
+-module(oprga_test).
+
+-export([confirm/0]).
+
+-include_lib("eunit/include/eunit.hrl").
+-define(HARNESS, (rt_config:get(rt_harness))).
+
+confirm() ->
+    [Nodes] = rt:build_clusters([3]),
```

Comment:
Maybe add these lines like was done in https://github.com/SyncFree/antidote/pull/143 to speed up the tests

    NumVNodes = rt_config:get(num_vnodes, 8),
    rt:update_app_config(all,[
        {riak_core, [{ring_creation_size, NumVNodes}]}
    ]),

Also you should probably add the lines to wait until the ring is converged since we don't have the handoff stuff implemented.